test/strategies/kandel/abstract/CoreKandel.t.sol

```diff
@@ -1,256 +1,20 @@
 // SPDX-License-Identifier:	AGPL-3.0
 pragma solidity ^0.8.10;
 
-import {IERC20} from "mgv_src/IERC20.sol";
-import {IMangrove} from "mgv_src/IMangrove.sol";
-import {MgvStructs, MgvLib} from "mgv_src/MgvLib.sol";
-import {OfferType} from "mgv_src/strategies/offer_maker/market_making/kandel/abstract/TradesBaseQuotePair.sol";
-import {CoreKandel, TransferLib} from "mgv_src/strategies/offer_maker/market_making/kandel/abstract/CoreKandel.sol";
-import {GeometricKandel} from "mgv_src/strategies/offer_maker/market_making/kandel/abstract/GeometricKandel.sol";
-import {KandelLib} from "mgv_lib/kandel/KandelLib.sol";
-import {console} from "forge-std/Test.sol";
-import {MangroveTest} from "mgv_test/lib/MangroveTest.sol";
-import {MgvReader} from "mgv_src/periphery/MgvReader.sol";
-import {AbstractRouter} from "mgv_src/strategies/routers/AbstractRouter.sol";
-import {AllMethodIdentifiersTest} from "mgv_test/lib/AllMethodIdentifiersTest.sol";
-
-abstract contract CoreKandelTest is MangroveTest {
-  address payable maker;
-  address payable taker;
-  GeometricKandel kdl;
-  uint8 constant STEP = 1;
-  uint initQuote;
-  uint initBase = 0.1 ether;
-  uint globalGasprice;
-  uint bufferedGasprice;
-
-  event Mgv(IMangrove mgv);
-  event Pair(IERC20 base, IERC20 quote);
-  event NewKandel(address indexed owner, IMangrove indexed mgv, IERC20 indexed base, IERC20 quote);
-  event SetGeometricParams(uint spread, uint ratio);
-  event SetCompoundRates(uint compoundRateBase, uint compoundRateQuote);
-  event SetLength(uint value);
-  event SetGasreq(uint value);
-  event Credit(IERC20 indexed token, uint amount);
-  event Debit(IERC20 indexed token, uint amount);
-  event PopulateStart();
-  event PopulateEnd();
-  event RetractStart();
-  event RetractEnd();
-  event LogIncident(
-    IMangrove mangrove,
-    IERC20 indexed outbound_tkn,
-    IERC20 indexed inbound_tkn,
-    uint indexed offerId,
-    bytes32 makerData,
-    bytes32 mgvData
-  );
-
-  // sets base and quote
-  function __setForkEnvironment__() internal virtual {
-    // no fork
-    options.base.symbol = "WETH";
-    options.quote.symbol = "USDC";
-    options.quote.decimals = 6;
-    options.defaultFee = 30;
-    options.gasprice = 40;
-
-    MangroveTest.setUp();
-  }
+import "./KandelTest.t.sol";
 
+abstract contract CoreKandelTest is KandelTest {
```

Comment:
To me it seems a bit weird to have CoreKandleTest derive from KandelTest. Should it be AbstractKandelTest? But it is not very important :)

Reply:
yes it's abstract. But we use the same convention for `MangroveTest` so I'll keep it for now